packages/cli/package.json

```diff
@@ -37,6 +37,7 @@
     "@types/inquirer": "^6.5.0",
     "@types/mustache": "^0.8.32",
     "@types/rewire": "^2.5.28",
+    "faker": "^4.1.0",
```

Comment:
I would say that `faker` was already accessible for the cli package.

Reply:
I wasn't able to import the package from cli without installing it

Reply:
Removed the package and worked perfectly, updated the package.json, thanks @glammers1